dictionaryutils/__init__.py

```diff
@@ -239,3 +239,19 @@ def resolve_schema(self, obj, root):
             return [self.resolve_schema(item, root) for item in obj]
         else:
             return obj
+
+    def allow_nulls(self):
+        for key, value in self.schema.items():
+            try:
+                try:
+                    required = value["required"]
+                except KeyError:
+                    required = []
```

Comment:
```
try:
    required = value["required"]
except KeyError:
    required = []
```
can be simplified to `required = value.get("required", [])`

Reply:
Yes, that is much better